scripts/linters/general_purpose_linter.py

```diff
@@ -39,6 +39,7 @@
     'core/tests/release_sources/tmp_unzip.zip', 'scripts/linters/test_files/*',
     'core/tests/release_sources/tmp_unzip.tar.gz',
     'core/templates/combined-tests.spec.ts',
+    'core/templates/css/oppia-material.css',
```

Comment:
Why we need this here?

Reply:
I added it here to be ignored by the linter because it is a generated CSS file and it was raising a lot of lint issues.